spec/features/admin_views_attendence_spec.rb

```diff
@@ -42,6 +45,36 @@
     end
   end
 
+  scenario "admin sees total number of active students" do
+    admin = FactoryGirl.create(:user, role: "admin")
+    FactoryGirl.create(:user_with_answered_quiz)
+    FactoryGirl.create(:user_with_answered_quiz)
+
+    sign_in(admin)
+
+    click_link 'View Attendance'
+
+    within(".columns.medium-6:first-child") do
+      expect(page).to have_content("Present")
+      expect(page).to have_content("Total: 2")
+    end
+  end
+
+  scenario "admin sees total number of active students" do
+    admin = FactoryGirl.create(:user, role: "admin")
```

Comment:
I believe there is an `:admin` factory that should get this done for us. If not we should create one.